### PR TITLE
[W5][W09-3]Phua Yue Jun

### PR DESCRIPTION
**Pull request by whyjayxp**
Add sort command that lists the persons in alphabetical order (ready for review)

### CODE REVIEW

**Review by beesaycheese — COMMENTED** [review]
Commit messages are short and comprehensive. 
Systematic and neat categorization of the commits.
Clear OOP style that follows the required coding standards.
All the needed individual components for Project Preparation 2KLoC are achieved.
Overall, good job! :)

File: src/seedu/addressbook/commands/SortCommand.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * Lists all persons in the address book to the user.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all persons in the address book as a list with index numbers in alphabetical order.\n"
+            + "Example: " + COMMAND_WORD;
+
```
Comment (marlenekoh) [inline src/seedu/addressbook/commands/SortCommand.java:18]: Extra line break here

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 2]
```diff
@@ -129,6 +129,14 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sorts the persons in the list in alphabetical order and returns itself.
+     */
+    public UniquePersonList sort() {
+        internalList.sort(new Person.NameComparator());
```
Comment (marlenekoh) [inline src/seedu/addressbook/data/person/UniquePersonList.java:136]: Try not to modify the original data, instead, modify a copy (: This gives more freedom to keep track of some properties of the original `UniquePersonList` and implement more sort commands in the future - e.g. sort by recently added

File: src/seedu/addressbook/parser/Parser.java [hunk 3]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (marlenekoh) [inline src/seedu/addressbook/parser/Parser.java:14]: Wildcard imports violate the coding standard. Please use single imports and change your settings in Intellij: `Settings` > `Editor` > `Code Style` > `Java` > `Imports` and set both class count and names count to 99 (or any large number)

**Review by marlenekoh — COMMENTED** [review]
@whyjayxp Good job updating your tests and documentation! Especially nice to see that you've tried to create a JUnit test for your command 👍 

Please close this PR after reading the review comments.

**Pull request closed without merging**